gradle/libs.versions.toml

```diff
@@ -0,0 +1,70 @@
+[versions]
+assertj = "3.24.2"
+aws = "2.20.18" # this is in mapping with iceberg repo.
+checkstyle = "10.9.3"
+ecs = "3.3.2"
+errorprone = "2.18.0"
+errorproneSlf4j = "0.1.18"
+googleJavaFormat = "1.16.0"
+guava = "31.1-jre"
+hadoop = "2.7.3" # this is in mapping with iceberg repo.
+hive = "2.3.8" # this is in mapping with iceberg repo.
+iceberg = "1.2.1"
+immutables = "2.9.3"
+jacoco = "0.8.9"
+jandex = "3.1.0"
+junit = "5.9.2"
+logback = "1.4.5"
+logcaptor = "2.8.0"
+mysqlDriver = "8.0.32"
+nessie = "0.57.0"
+nessieRunner = "0.29.0"
+picocli = "4.7.2"
+shadowPlugin = "8.1.1"
+slf4j = "1.7.36"
+
+[libraries]
+assertj = { module = "org.assertj:assertj-core", version.ref = "assertj" }
+aws-sdk-apache-client = { module = "software.amazon.awssdk:apache-client", version.ref = "aws" }
+aws-sdk-auth = { module = "software.amazon.awssdk:auth", version.ref = "aws" }
+aws-sdk-dynamo = { module = "software.amazon.awssdk:dynamodb", version.ref = "aws" }
+aws-sdk-glue = { module = "software.amazon.awssdk:glue", version.ref = "aws" }
+aws-sdk-kms = { module = "software.amazon.awssdk:kms", version.ref = "aws" }
+aws-sdk-lakeformation = { module = "software.amazon.awssdk:lakeformation", version.ref = "aws" }
+aws-sdk-sts = { module = "software.amazon.awssdk:sts", version.ref = "aws" }
+aws-sdk-s3 = { module = "software.amazon.awssdk:s3", version.ref = "aws" }
+aws-sdk-url-connection-client = { module = "software.amazon.awssdk:url-connection-client", version.ref = "aws" }
+checkstyle = { module = "com.puppycrawl.tools:checkstyle", version.ref = "checkstyle" }
+errorprone-annotations = { module = "com.google.errorprone:error_prone_annotations", version.ref = "errorprone" }
+errorprone-core = { module = "com.google.errorprone:error_prone_core", version.ref = "errorprone" }
+errorprone-slf4j = { module = "jp.skypencil.errorprone.slf4j:errorprone-slf4j", version.ref = "errorproneSlf4j" }
+ecs-bundle = { module = "com.emc.ecs:object-client-bundle", version.ref = "ecs" }
```

Comment:
Commercially licensed, NOT ASF2 licensed.
Remove it from this PR.
I do not wanna go down the rabbit hole and discuss this issue in this PR.

Reply:
Apache Iceberg (Apache 2.0 license) is also dependent on this for compile time `iceberg-ecs` module.

https://github.com/apache/iceberg/blob/b756a5da3559be9deee9cdc674c4bf36ac291231/build.gradle#L820

So, I thought it is compatible. 

I will remove it. If the user gets a ClassNotFound while migrating to ECS catalog, they might have to add this jar in the classpath.